utils/rust-setup.js

```diff
@@ -0,0 +1,122 @@
+const chalk = require('chalk');
+const os = require('os');
+const readline = require('readline');
+const sh = require('shelljs');
+
+// script from https://rustup.rs/ with auto-accept flag "-y"
+const installRustupScript = "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y";
+const updatePath = 'source $HOME/.cargo/env';
+const addWasm32TargetScript = "rustup target add wasm32-unknown-unknown";
+// We should update PATH in the same script because every new Bash scripts are executed in a separate shell
+const updatePathAndAddWasm32TargetScript = updatePath + ' && ' + addWasm32TargetScript;
+
+const isWindows = os.platform() == 'win32'
+
+function isRustupInstalled() {
+    console.log(chalk`Checking if {bold rustup} is installed...`);
+    const isInstalled = sh.exec('rustup --version &> /dev/null').code == 0;
+    console.log(chalk`{bold rustup} is`, isInstalled ? 'installed' : 'not installed');
+    return isInstalled;
+}
+
+function isWasmTargetAdded() {
+    const addedTargets = sh.exec('rustup target list --installed').stdout;
```

Comment:
In `isRustupInstalled` we `console.log` at the beginning to say "Checking if **rustup** is installed..."

We also redirect stdout to /dev/null

Should we do those same things here?

Reply:
We are using `stdout` from `rustup target list --installed`, so we can not send it to `/dev/null`.

I have added an additional console.log, now it looks like this:

`Checking installed Rust targets..`
`wasm32-unknown-unknown`
`x86_64-unknown-linux-gnu`
`wasm32-unknown-unknown target  already added`